enterprise/releases/2024-10-0.mdx

```diff
@@ -0,0 +1,92 @@
+  - With this change, it'll now respect the time selected in the date picker.
+  - In addition to this, introduce the pagination and also ability to search by the plan fingerprint to the list.
+- Remove compatibility layer for packed stats rollout <!-- 4050 -->
+  - Release 2024.04.0 contained dual-write logic for internal stats tables to migrate to a new packed structure without requiring an expensive database migration or otherwise dropping historical data. This release drops the old table structure. As long as you have been running release 2024.04.x for 30 days, no data will be lost with this release.
+- Install Docs: Keep same generated password when moving install between steps <!-- 3933 -->
+- Fix handling of single-line SQL comments in formatted queries <!-- 3979 -->
+- Fix snapshot upload failures caused by file size limit <!-- 3986 -->
+- Rename memory stats that belong to the OS to avoid confusion with Postgres `shared_buffers` <!-- 3988 -->
+- Per-table VACUUM information: Fix rendering bug when autovacuum_vacuum_insert_threshold is 0 <!-- 4087 -->
+- Schema Statistics: Increase width of schema name column in table listing <!-- 4183 -->
+- Remove support for collector versions below 0.40.0 <!-- 4160 -->
+- VACUUM Advisor: Hide negative bloat estimates <!-- 4219 -->
+  - In some situations, bloat can be estimated as negative due to limitations of the bloat estimation heuristics. Hide estimates from these cases to avoid confusion.
+- Improve date range selection on small screens <!-- 4229 -->
+- EXPLAIN: Fix date range selector menu layout issue on smaller screens <!-- 4239 -->
+- Update a duplicate snapshot warning logic in Server Settings page <!-- 4240 -->
+  - Previously, this was wrongly providing a warning when the test snapshot is executed recently. This is now fixed.
+- System Stats: Fix CPU core count calculation <!-- 4244 -->
+  - We previously multiplied the CPU logical core count by the number of\nsockets to determine this number, but that's not the commonly expected\nway of calculating this. Also rename the field to "Logical Cores" to\nclarify the meaning.
+ LDAP Authentication: Require valid SSL certificate for LDAP server if secure connection is specified <!-- 4300 -->
+- Schema Statistics: Show notice when the column stats are not available for the table due to outdated stats <!-- 4311 -->
+  - When the statistics in pg_stats were out of sync with the current table structure, such as when columns have been added or removed since the last statistics collection, they were simply not shown previously. With this change, show notification so that ANALYZE can be run to solve the issue.
+- EXPLAIN: Tweak plan fingerprinting to improve detection of partition indexes <!-- 4304 -->
+  - This will result in the same fingerprint for substantially identical\nquery plans that were previously fingerprinted distinctly.
```

Comment:
```suggestion
  - This will result in the same fingerprint for substantially identical query plans that were previously fingerprinted distinctly.
```